SharedSources/PresentationTheme.swift

```diff
@@ -79,6 +76,15 @@ extension Notification.Name {
     }
 }
 
+@objcMembers public class Typography: NSObject {
+    
+    public let tableHeader: UIFont
+    
+    public init(tableHeader: UIFont) {
+        self.tableHeader = tableHeader
```

Comment:
Is there a specific reason why we would want this class to be `public`?

Reply:
Hmmm I guess it's unnecessary